retrievaltool/src/main/java/org/duracloud/retrieval/source/DuraStoreSpecifiedRetrievalSource.java

```diff
@@ -55,6 +68,81 @@ public DuraStoreSpecifiedRetrievalSource(ContentStore store,
         }
 
         this.specifiedContentIds = specifiedContentIds;
+        this.reviewSpecifiedContentIdsForChunkedContent(singleSpaceList);
+    }
+
+    protected void reviewSpecifiedContentIdsForChunkedContent(List<String> singleSpaceList) {
+        log.debug("enter remediateSpecifiedContentIds()");
+
+        List<String> retrievalContentIds = new ArrayList<String>();
+        while (specifiedContentIds.hasNext()) {
+            String temp = specifiedContentIds.next();
+            retrievalContentIds.add(temp);
+        }
+        log.debug("total contentIds in list-file: " + retrievalContentIds.size());
+
+        List<String> retrievalSpaceContentIds = new ArrayList<String>();
+
+        Iterator<String> retrievalSpaceIds = verifySpaceIds(singleSpaceList);
+        if (retrievalSpaceIds.hasNext()) {
+            String currentRetrievalSpaceId = retrievalSpaceIds.next();
+            log.debug("searching for contentIds in space: " + currentRetrievalSpaceId);
+            try {
+                InputStream manifest = contentStore.getManifest(currentRetrievalSpaceId, ManifestFormat.TSV);
+                ManifestFormatter formatter = new ManifestFormatterFactory().create(ManifestFormat.TSV);
+                String header = formatter.getHeader();
+                BufferedReader reader = new BufferedReader(new InputStreamReader(manifest));
+                String line = null;
+                ManifestItem item = null;
+
+                try {
+                    while ((line = reader.readLine()) != null) {
+                        // ignore any whitespace
+                        if (line.trim().length() == 0) {
+                            continue;
+                        }
+
+                        // ignore header line
+                        if (line.equals(header)) {
+                            continue;
+                        }
+
+                        try {
+                            item = formatter.parseLine(line);
+                        } catch (ParseException e) {
+                            throw new IOException(e);
+                        }
+
+                        String contentId = item.getContentId();
+
+                        // search for chunks or chunk manifests for contentId
+                        String tempContentId = contentId;
+                        if (contentId.endsWith(ChunksManifest.manifestSuffix)) {
+                            tempContentId = StringUtils.removeEnd(contentId, ChunksManifest.manifestSuffix);
+                        } else if (contentId.contains(ChunksManifest.chunkSuffix)) {
+                            tempContentId = contentId.substring(0, contentId.indexOf(ChunksManifest.chunkSuffix));
+                        }
+
+                        if (retrievalContentIds.contains(tempContentId)) {
+                            log.debug("found chunked content for contentId in list-file: " + contentId);
+                            retrievalSpaceContentIds.add(contentId);

```

Comment:
This appears to be adding all of the chunks to the retrieval list along with the chunk manifests. Looking at `DuraStoreStitchingRetrievalSource.doGetContent()`, it looks like this will result in a logged error and a thrown exception for each chunk file in the list. Why not just ignore the chunks and add the manifests to the list?